content/en/user-guide/ci/ci-analytics/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "CI Analytics"
+linkTitle: "CI Analytics"
+weight: 1
+description: Get insights into your continuous integration (CI) builds with LocalStack CI Analytics
+---
+
+## Introduction
+
+CI Analytics is a feature of LocalStack Web Application that allows users to get insights into their CI builds. CI Analytics unifies additional features, such as Cloud Pods & Stack Insights, and augments them with an integrated view of CI builds to provide users with insights and facilitate debugging. The features include:
+
```

Comment:
We should add a note here that this is in Preview/Beta. Let's tackle that right after the release.. 👍